segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -417,11 +425,13 @@ private ServiceConfig(TypedProperties properties) throws ConfigurationException
                 .port(properties.getInt(REST_LISTENING_PORT))
                 .tlsEnabled(properties.getBoolean(ENABLE_TLS))
                 .tlsProtocolVersion(TLSProtocolVersion.parse(properties.get(TLS_PROTOCOL_VERSION)))
-                .keyFilePath(properties.get(KEY_FILE))
-                .keyFilePasswordPath(properties.get(KEY_PASSWORD_FILE))
+                .keyFilePath(properties.get(REST_KEYSTORE_FILE))
+                .keyFilePasswordPath(properties.get(REST_KEYSTORE_PASSWORD_FILE))
                 .build();
         this.restServerEnabled = properties.getBoolean(REST_LISTENING_ENABLE);
         this.healthCheckInterval = Duration.ofSeconds(properties.getInt(HEALTH_CHECK_INTERVAL_SECONDS));
+        this.restKeyStoreFile = properties.get(REST_KEYSTORE_FILE);
```

Comment:
Why do we need to keep these properties twice? (Once here and once in `restServerConfig`. Since these are used only in the latter, let's get rid of these two extra fields.

Reply:
They are used again to initialize the class variables restkeyStoreFile and restKeyStorePasswordFile of Service Class

Reply:
They aren't used anywhere from what I see. You declared them in this PR and outside of this class they aren't used anywhere else. I was referring to the class variables - why do you need them when they're already accesisible via the `RestConfig`?

Reply:
Fixed. Yes they were never used except in this class. Removed them. Thanks.